core/README.md

```diff
@@ -43,7 +43,7 @@
 | [CAP-0015](cap-0015.md) | Bump Fee Transactions | OrbitLens | Draft |
 | [CAP-0016](cap-0016.md) | Cosigned assets: NopOp and COAUTHORIZED_FLAG | David Mazières | Draft |
 | [CAP-0017](cap-0017.md)| Update LastModifiedLedgerSeq If and Only If LedgerEntry is Modified | Jonathan Jove | Draft |
-| [CAP-0018](cap-0018.md)| Fine-Grained Control of Authorization | Jonathan Jove | FCP: Acceptance |
+| [CAP-0018](cap-0018.md)| Fine-Grained Control of Authorization | Jonathan Jove | Accepted |
```

Comment:
Hey @jonjove could you remove this for now until next week, given you had to address more feedback? Would prefer the FCP process to be a full quiet week.

Reply:
Done.